frontend/src/screens/achievements.tsx

```diff
@@ -0,0 +1,419 @@
+                        lineHeight: '28px',
+                        textAlign: 'center',
+                    }}
+                >
+                    {study?.learningPath?.label}
+                </Text>
+                <Text
+                    size="xs"
+                    color="dimmed"
+                    mb={10}
+                    style={{
+                        fontFamily: 'System-ui',
+                        fontSize: 12,
+                        lineHeight: '18px',
+                        textAlign: 'center',
+                        color: colors.gray70,
+                    }}
+                >
+                    {`${completedStudies} of ${study?.learningPath?.studies.length}`}
+                </Text>
+                <Button
+                    onClick={(e) => handleButtonClick(e, study.learningPath.badgeId)}
+                    style={{
+                        width: '200px',
+                        height: '30px',
+                        padding: '8px 20px',
+                        gap: '30px',
+                        borderRadius: '4px',
+                        border: `1px solid ${colors.purple}`,
+                        backgroundColor: 'white',
+                        color: colors.purple,
+                        fontSize: '14px',
+                        fontWeight: 'bold',
+                        display: 'flex',
+                        alignItems: 'center',
+                        justifyContent: 'center',
+                    }}
+                >
+                    {buttonText}
+                </Button>
+            </Box>
+        </Box>
+    );
+};
+
+const TabButton = ({
+    label,
+    isActive,
+    onClick,
+}: {
+    label: string;
+    isActive: boolean;
+    onClick: () => void;
+}) => (
+    <Button
+        variant="subtle"
+        color={isActive ? colors.blue : 'black'}
+        onClick={onClick}
+        style={{
+            padding: '8px 16px',
+            fontWeight: 500,
+            fontSize: '16px',
+            textTransform: 'uppercase',
+            borderBottom: isActive ? `2px solid ${colors.purple}` : 'none',
+            borderRadius: 0,
+        }}
+    >
+        {label}
+    </Button>
+);
+
+const Achievements = () => {
+    const [selectedTab, setSelectedTab] = useState<'Badges' | 'Points'>(
+        'Badges'
+    );
+    const [selectedStudy, setSelectedStudy] = useState(null);
+    const [badgeDetail, setBadgeDetail] = useState(null);
+
+    const DATA = useParticipantStudies();
+    const studies = DATA.studies;
+
+    const handleTabClick = (tab: any) => setSelectedTab(tab);
+    const handleBadgeClick = (study: any) => {
+        setBadgeDetail(study);
+    };
+    const handleCloseDetail = () => {
+        setBadgeDetail(null);
+    };
+    const handleCloseStudyDetails = () => {
+        setSelectedStudy(null);
+    };
+    const handleStudySelect = (study: any) => setSelectedStudy(study);
+
+    const renderContent = () => {
```

Comment:
did you consider using the [mantine tabs component?](https://mantine.dev/core/tabs/)